stdlib/2and3/lib2to3/pgen2/parse.pyi

```diff
@@ -0,0 +1,30 @@
+# Stubs for lib2to3.pgen2.parse (Python 3.6)
+
+from typing import Any, Dict, List, Optional, Sequence, Set, Text, Tuple
+
+from lib2to3.pgen2.driver import _Convert, _RawNode
+from lib2to3.pgen2.grammar import Grammar, _DFAS
+from lib2to3.pytree import _NL
+
+_Context = Sequence[Any]
+
+class ParseError(Exception):
+    msg: Text
+    type: int
+    value: Optional[Text]
+    context: _Context
+    def __init__(self, msg: Text, type: int, value: Optional[Text], context: _Context) -> None: ...
+
+class Parser:
+    grammar: Grammar
+    convert: _Convert
```

Comment:
Hm, this defaults to `lambda grammar, node: node`, which doesn't appear to satisfy your definition of `_Convert`. The docstring says that the function must return an abstract syntax tree node and "An abstract syntax tree node may be anything; this is entirely up to the converter function.", so I think the type for the convert function here should be `Callable[[Grammar, _RawNode], Any]`.